recipe/migrations/osx_arm64.txt

```diff
@@ -959,3 +959,4 @@ iteration_utilities
 hmmlearn
 arm_pyart
 stockwell
+rubin_sim
```

Comment:
```suggestion
rubin-sim
```

Reply:
I think it needs to be this?

Reply:
I was following along with https://github.com/lsst/rubin_sim#conda-installation

Reply:
Right. This is the name of the feedstock, not the package iirc